horreum-web/src/domain/runs/JsonPathSearchToolbar.tsx

```diff
@@ -207,9 +187,9 @@ function execQuery(
     switch (type) {
         case "js":
```

Comment:
Following the same approach below
```suggestion
        case JS_QUERY_LOCAL:
```